file_manager/run_file.py

```diff
@@ -0,0 +1,96 @@
+import threading
+import os
+import traceback
+
+import pexpect
+
+from helpers.logger import create_logger
+
+
+FILE_PATH = '/home/username/Code_Docs/running_files/'
```

Comment:
That field is different on different machines. Can we find this path? or may be create environment variable?

Reply:
task for the future

Reply:
ok